src/session.ts

```diff
@@ -610,6 +610,11 @@ export class SessionManager implements Middleware {
                 });
         };
 
+        // When Terminal Shell Integration is enabled, we pass the path to the script that the server should execute.
+        // Passing an empty string implies integration is disabled.
+        const shellIntegrationEnabled = vscode.workspace.getConfiguration("terminal.integrated.shellIntegration").get<boolean>("enabled");
+        const shellIntegrationScript = path.join(vscode.env.appRoot, "out", "vs", "workbench", "contrib", "terminal", "browser", "media", "shellIntegration.ps1");
```

Comment:
@Tyriar I'm taking a hard dependency on this relative-to-app-root path. You all have too, but just internally as far as I can tell. Trust me, it was no good going down the road of passing `process.argv0` and trying to use `code --locate-blah-blah`.

Reply:
Good to know, chances of that path changing are very slim anyway